package.json

```
@@ -2,7 +2,7 @@
   "name": "fabric8-analytics",
   "displayName": "Dependency Analytics",
   "description": "Insights about your application dependencies: Security, License compatibility and AI based guidance to choose appropriate dependencies for your application.",
-  "version": "0.0.14-2020-07-01-2",
+  "version": "0.0.14-2020-07-07",
```

Please follow the convention. The last number so far has been used to indicate the no of test builds done so far for 0.0.14 beta testing.
```suggestion
  "version": "0.0.14-2020-07-07-3",
```